packages/web-twig/src/Resources/components/Toast/stories/ToastDynamicToastQueue.twig

```diff
@@ -51,22 +97,19 @@
 </form>
 <!-- Toast and visual keyboard demo: end -->
 
-<!-- Toast show and hide demo: start -->
-<fieldset style="border: 0;">
-    <legend class="mb-500">Show the toast prepared in the DOM:</legend>
-    <Button
-        data-spirit-toggle="toast"
-        data-spirit-target="#my-hidden-dismissible-toast"
-        onclick="this.setAttribute('disabled', 'disabled')"
-        aria-expanded="false"
-    >
-        Show the hidden toast
-    </Button>
-</fieldset>
-<!-- Toast show and hide demo: end -->
+<Icon name="check-plain" isSymbol />
+<Icon name="danger" isSymbol />
+<Icon name="info" isSymbol />
+<Icon name="warning" isSymbol />
 
 <Toast id="toast-example">
```

Comment:
Too many toasts here! 🍞 

- The **second** one with ID `template-toast` is outside of `<template>` and should be deleted.
- The one with ID `my-hidden-dismissible-toast` is to be deleted as it is already present in the previous demo (aka "show hidden in the DOM").